DEPS.bzl

```diff
@@ -3401,8 +3401,9 @@ def go_deps():
         name = "com_github_tikv_client_go_v2",
         build_file_proto_mode = "disable_global",
         importpath = "github.com/tikv/client-go/v2",
-        sum = "h1:pZoPlKWCecxJKL8oRq/se71RTljYDrQlZQ2NzKkMYi0=",
-        version = "v2.0.1-0.20220830073839-0130f767386c",
+        replace = "github.com/ekexium/client-go/v2",
```

Comment:
Don't forget to re-generate bazel files.

Reply:
Always 😢